### DISCUSSION TITLE
How can i delete 'dead' pages ?

### DISCUSSION THREAD

**Discussion by samhk222** [zim-desktop-wiki/zim-desktop-wiki#2551] [General]
![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/58010/35b80234-14df-40bd-8334-d9cf5a5b80d2)

I'm a heavy user of zim, it's my 'always opened app' since 2020 but sometimes when i paste something wrong (i think that's the case) it creates some ghost pages lije this one attached here.

How can i remove those automatically ? 

Thanks! ❤️

**Comment by roland-ruedenauer** [comment]
Please read the section about the use of [CamelCase](https://zim-wiki.org/manual/Usage/Getting_Started.html#use-camelcase).  You can disable the automtic link creation in the Zim settings.

**Reply by introt** [reply]
See also: #834
